BibBuddy/app/src/main/java/de/bibbuddy/BookNotesView.java

```diff
@@ -103,14 +104,13 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
       bookId = bundle.getLong(LibraryKeys.BOOK_ID);
     }
 
-    // TODO model instead of dao
-    bookDao = bookNotesViewModel.getBookDao();
-    noteDao = bookNotesViewModel.getNoteDao();
+    bookModel = new BookModel(requireContext(), getShelfId());
+    noteModel = new NoteModel(requireContext());
 
-    String fileName = (bookDao.findById(bookId).getTitle()
-        + bookDao.findById(bookId).getPubYear())
+    String fileName = (bookModel.getBookById(bookId).getTitle()
+        + bookModel.getBookById(bookId).getPubYear())
```

Comment:
```suggestion
    Book book = bookModel.getBookById(bookId);
    String fileName = (book.getTitle() + book.getPubYear())
```

Please wait until my review is done before you commit something. Also I would appreciate it if you could put all the changes into a single commit so that I don't have to look at all commits separately.